kamon-metrics/src/main/scala/com/ovoenergy/effect/Metrics.scala

```diff
@@ -24,9 +25,15 @@ object Metrics {
   /**
    * An instance of metrics for a Sync[F] which wraps the underlying calls
    */
-  def syncKamonMetrics[F[_]: Sync: FlatMap]: Metrics[F] =
-    metric =>
-      FlatMap[F].map(Sync[F].delay(Kamon.metrics.counter(metric.name, metric.tags)))(
+  implicit def syncMetrics[F[_]: Sync]: Metrics[F] = new Metrics[F] {
+    override def counter(metric: Metric): F[Long => F[Unit]] =
```

Comment:
`override`? how did this get passed tom?

Reply:
Doesn't hurt, but I'll ditch it